backend/app/services/readers/kinds.py

```
@@ -0,0 +1,271 @@
+    base = KindReader()
+
+    if settings.SERVICE_EXTENSION:
+        try:
+            import importlib
+
+            ext = importlib.import_module(f"{settings.SERVICE_EXTENSION}.kinds")
+            result = ext.wrap(base)
+            if result:
+                logger.info("Kind reader extension loaded")
+                return result
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)